index.html

```diff
@@ -12,7 +12,15 @@
 <body>
   <div class="body-container">
     <h2 class="main-title">Front-End Tools</h2>
-    <h2 class="main-title">Back-End Tools</h2>
+    <div class="tech-border">
+      <div class="tech-container">
+        <img class="tech-image" src="./assets/react_icon.svg" alt="react">
+        <h2 class="tech-title">ReactJS</h2>
+        <span class="tech-title">Release: 2013</span>
+        <p class="tech-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+      </div>
```

Comment:
Check that the `tech title` and `tech release` date are not equal size.
<img width="223" alt="captura de pantalla 2018-05-31 a la s 14 23 18" src="https://user-images.githubusercontent.com/36163179/40800365-368b7906-64de-11e8-90ba-708b3b8f1fec.png">

So i would create another class for the release date like `tech-release` in order to be a little bit smaller.
Maybe `tech description` and `tech release` can have the same font only differing by the font weight.